clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -2952,6 +2952,19 @@ void prepareTypeConverter(mlir::LLVMTypeConverter &converter,
   converter.addConversion([&](mlir::cir::DoubleType type) -> mlir::Type {
     return mlir::FloatType::getF64(type.getContext());
   });
+  converter.addConversion([&](mlir::cir::FP80Type type) -> mlir::Type {
+    return mlir::FloatType::getF80(type.getContext());
```

Comment:
Please add testcases that cover all conversions to LLVM. You might need to look at llvm tests for references on how to require a target when writing a test (you have ppc and x86 specifics here).

Reply:
I have removed `!cir.f128` and `!cir.ppc_doubledouble` from this PR because their targets are not yet supported by CIR. See my latest comment.

Reply:
perfect, thanks